### DISCUSSION TITLE
When Versioning scheme "path" is selected, changes in minor version causes breaking API change (new URL)

### DISCUSSION THREAD

**Discussion by markbriggs-ms** [Azure/api-management-developer-portal#2596] [Feature request]
Version has to be stated in the Azure Portal form-based editor when creating a new minor version (as per below)

![image](https://user-images.githubusercontent.com/44032869/169275302-0d516721-9bec-45a3-b9a9-4e793db1d5f6.png)

Creating a minor version then impacts the URL a consumer would use ... e.g.

http://someapi.azure-api.net/sampleapi/**v1.0.16**/investor-pricing/...... etc

Customer would like to only have the major version reflected in the path e.g.

http://someapi.azure-api.net/sampleapi/v1/investor-pricing/...... etc

minor/patch version should be kept so it can get exported as version tag in the openapi spec file.

This is more of a feature request than a bug.  Note: passing version as header or query parameter would not be suitable for the customers setup.

**Comment by ghost** [comment]
@markbriggs-ms, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@markbriggs-ms, thank you for requesting this feature.

**Comment by dzemp** [comment]
Hi @azaslonov  any update on this issue / feature request?
